README.md

```diff
@@ -176,6 +176,15 @@ mappings:
     provider: "$2"
     outcome: "$3"
     job: "${1}_server"
+  quantiles:           # Optionally configure quantiles for your summaries
```

Comment:
Ooops sorry I didn't catch this earlier – this is in the example for _histograms_.

What happens with quantiles in a `timer_type: histogram` match? Does that do anything?

I'd rather break this out into a separate example before this one, with something like "By default, timers are recorded as a summary. Optionally you can configure the [quantiles](https://prometheus.io/docs/practices/histograms/#quantiles)."